libs/common-services/src/main/java/uk/gov/di/ipv/core/library/domain/UserIdentity.java

```diff
@@ -64,5 +72,6 @@ public UserIdentity(
         this.sub = sub;
         this.vot = vot;
         this.vtm = vtm;
+        this.exitCode = exitCode;
```

Comment:
(In case you were worried, I checked with orchestration and they've asserted that there won't be an issue if we start sending this before they're handling it)

Reply:
They were going to be my next port of call! Good to know.